### PR TITLE
Generation

### PR DESCRIPTION
**Pull request by yadongyaly**
## Description

Adding custom cmdlets for New-AzDataMigrationLoginsMigration. Add two files in custom folder : New-AzDataMigrationLoginsMigration.ps1 and LoginMigrations.ps1

It has ran .\build-module.ps1 already. This PR also fixed a typo in the example folder : Get-AzDataMigrationPerformanceDataCollection.md file has a typo. 

After running .\build-module.ps1, a lot of files in doc folder were updated, but it seems only descriptions. 

The new custom cmdlet has been tested locally by using .\run-module.ps1

## Checklist

- [ ] Check this box to confirm: **I have read the [_Submitting Changes_](../blob/main/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/main/CONTRIBUTING.md) and reviewed the following information:**

* **SHOULD** select appropriate branch. Cmdlets from Autorest.PowerShell should go to [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch. 
* **SHOULD** make the title of PR clear and informative, and in the present imperative tense. 
* **SHOULD** update `ChangeLog.md` file(s) appropriately
    * For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    * A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header in the past tense. Add changelog in description section if PR goes into [`generation`](https://github.com/Azure/azure-powershell/tree/generation) branch.
    * Should **not** change `ChangeLog.md` if no new release is required, such as fixing test case only.
* **SHOULD** have approved design review for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) ([_Microsoft internal only_](../blob/main/CONTRIBUTING.md#onboarding)) with following situations
    * Create new module from scratch
    * Create new resource types which are not easy to conform to [Azure PowerShell Design Guidelines](../blob/main/documentation/development-docs/design-guidelines)
    * Create new resource type which name doesn't use module name as prefix
    * Have design question before implementation
* **SHOULD** regenerate markdown help files if there is cmdlet API change. [Instruction](../blob/main/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)
* **SHOULD** have proper test coverage for changes in pull request.
* **SHOULD NOT** introduce [breaking changes](../blob/main/documentation/breaking-changes/breaking-changes-definition.md) in Az minor release except preview version.
* **SHOULD NOT** adjust version of module manually in pull request

### CODE REVIEW

File: src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1 [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+        [System.String]
+        ${TargetSqlConnectionString},
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Optional. Location of CSV file of logins. Use only one parameter between this and listOfLogins.')]
+        [System.String]
+        ${CSVFilePath},
+
+        [Parameter(ParameterSetName='CommandLine', HelpMessage='Optional. List of logins in string format. If large number of logins need to be migrated, use CSV file option.')]
+        [System.String[]]
+        ${ListOfLogins},
```
Comment (wyunchi-ms) [inline src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1:45]: Please use singularity as parameter name. You can find a guideline [here](https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#singularity)

File: src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1 [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+.Description
+Migrate logins from the source Sql Servers to the target Azure Sql Servers.
+#>
+
+
+function New-AzDataMigrationLoginsMigration
+{
+    [OutputType([System.Boolean])]
+    [CmdletBinding(PositionalBinding=$false)]
+    [Microsoft.Azure.PowerShell.Cmdlets.DataMigration.Description('Migrate logins from the source Sql Servers to the target Azure Sql Servers.')]
```
Comment (wyunchi-ms) [inline src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1:27]: `New-AzDataMigrationLoginsMigration` Does not support ShouldProcess but the cmdlet verb New indicates that it should.

**Review by wyunchi-ms — APPROVED** [review]

**Comment by wyunchi-ms** [comment]
Please update the changelog base on this [PR](https://github.com/Azure/azure-powershell/pull/19608) @yadongyaly

**Comment by wyunchi-ms** [comment]
```csv
"DataMigration","New-AzDataMigrationLoginsMigration","1","1","Unbinded_Expression","5014","1","New-AzDataMigrationLoginsMigration 'loginname2' is not explicitly assigned to a parameter.","'loginname2'","Assign 'loginname2' explicitly to the parameter."
```
Please also fix the help document.

File: src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1 (hunk 1 above)
Comment (yadongyaly) [reply src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1]: > Please update the changelog base on this [PR](https://github.com/Azure/azure-powershell/pull/19608) @yadongyaly

Should I add change log on the "Comments" section of that PR?  I just added a comment for change log on your PR link

Comment (yadongyaly) [reply src/DataMigration/DataMigration.Autorest/custom/Cmdlets/New-AzDataMigrationLoginsMigration.ps1]: @wyunchi-ms wyunchi-ms

**Comment by yadongyaly** [comment]
> 



> ```
> "DataMigration","New-AzDataMigrationLoginsMigration","1","1","Unbinded_Expression","5014","1","New-AzDataMigrationLoginsMigration 'loginname2' is not explicitly assigned to a parameter.","'loginname2'","Assign 'loginname2' explicitly to the parameter."
> ```
> 
> Please also fix the help document.

Hi @wyunchi-ms , what does this mean? I could not find the file for this comment. Can you please help to give guidance about this fix? Thanks

**Pull request merged**